rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -912,11 +944,66 @@ void main_window::DecryptSPRXLibraries()
 				{
 					gui_log.error("Failed to create %s", new_path);
 				}
+
+				break;
 			}
-			else
+			else if (!invalid)
 			{
-				gui_log.error("Failed to decrypt %s", old_path);
+				// Allow the user to manually type KLIC if decryption failed
+
+				const std::string filename = old_path.substr(old_path.find_last_of(fs::delim) + 1);
+	
+				const QString hint = tr("Hint: KLIC (KLicense key) is a 16-byte long string. (32 hexadecimal characters)"
```

Comment:
since you already tried here.... why not tell the user that it failed ?

Like...
```
The provided KLIC did not work.

Hint: A KLIC (KLicense key) is a 16-byte long string (32 hexadecimal characters)
and is logged with some sceNpDrm* functions when the game/application which owns \"%0\" is running."
```

Reply:
Done.